stories/topics/proc-aws-generate-restore-data-file.adoc

```diff
@@ -2,7 +2,7 @@
 
 = Generating the restore data file
 
-The following commands create a directory, and populate it with an empty data template that, when completed, will be used during the backup. 
+The following commands create a directory, and populate it with an empty data template that, when completed, will be used during the restore. 
```

Comment:
data template that, when completed, is used during...

Reply:
fixed